src/main/viz/GenomeTrack.js

```diff
@@ -53,7 +53,7 @@ function renderGenome(ctx: DataCanvasRenderingContext2D,
         // We only push objects in the text case as it involves creating a
         // new object & can become a performance issue.
         // 0.5 = centered
-        ctx.fillText(letter, scale(1 + 0.5 + pos), height - 1);
+        ctx.fillText(letter, scale(1 + 0.5 + pos), height / 2);
```

Comment:
The argument changed was the y-coordinate. The previous y coordinate was height - 1, which was placing the text at the lower end of the canvas. By setting it to height / 2, the text was centered vertically.